test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -432,7 +442,7 @@ def test_successful_purchase_with_shipping_address
   end
 
   def test_successful_purchase_without_phone_number
```

Comment:
Good idea to include this! Perhaps we should include one that validates nothing unexpected happens if `payment_method.name` contains no data too?